src/main/java/org/openrewrite/java/migrate/apache/commons/lang/ApacheCommonsStringUtils.java

```diff
@@ -0,0 +1,357 @@
+        }
+
+        @AfterTemplate
+        String[] after(String s) {
+            return s.split(" ");
+        }
+    }
+
+    public static class StripEnd {
+        @BeforeTemplate
+        String before(String s) {
+            return StringUtils.stripEnd(s, "suffix");
+        }
+
+        @AfterTemplate
+        String after(String s) {
+            return s.endsWith("suffix") ? s.substring(0, s.lastIndexOf("suffix")) : s;
+        }
+    }
+
+    public static class Strip {
+        @BeforeTemplate
+        String before(String s) {
+            return StringUtils.strip(s);
+        }
+
+        @AfterTemplate
+        String after(String s) {
+            return s.trim();
+        }
+    }
+
+    //public static class LeftPad {
+    //    @BeforeTemplate
+    //    String before(String s) {
+    //        return StringUtils.leftPad(s, 5);
+    //    }
+
+    //    @AfterTemplate
+    //    String after(String s) {
+    //        return String.format("%" + 5 + "s", s);
+    //    }
+    //}
+
+    //public static class RightPad {
+    //    @BeforeTemplate
+    //    String before(String s) {
+    //        return StringUtils.rightPad(s, 5);
+    //    }
+
+    //    @AfterTemplate
+    //    String after(String s) {
+    //        return String.format("%" + (-5) + "s", s);
+    //    }
+    //}
+
+    public static class Join {
+        @BeforeTemplate
+        String before(String s) {
+            return StringUtils.join(s);
+        }
+
+        @AfterTemplate
+        String after(String s) {
+            return String.join(s);
+        }
+    }
+
+    public static class DeleteWhitespace {
+        @BeforeTemplate
+        String before(String s) {
+            return StringUtils.deleteWhitespace(s);
+        }
+
+        @AfterTemplate
+        String after(String s) {
+            return s.replaceAll("\\s+", "");
+        }
+    }
+
+    public static class EndsWithIgnoreCase {
+        @BeforeTemplate
+        boolean before(String s) {
+            return StringUtils.endsWithIgnoreCase(s, "suffix");
+        }
+
+        @AfterTemplate
+        boolean after(String s) {
+            return s.toUpperCase().endsWith("suffix".toUpperCase());
+        }
+    }
+
+    public static class IsAlphanumeric {
+        @BeforeTemplate
+        boolean before(String s) {
+            return StringUtils.isAlphanumeric(s);
+        }
+
+        @AfterTemplate
+        boolean after(String s) {
+            return s.matches("^[a-zA-Z0-9]*$");
+        }
+    }
+
+    public static class IsAlphanumericSpace {
+        @BeforeTemplate
+        boolean before(String s) {
+            return StringUtils.isAlphanumericSpace("^[a-zA-Z0-9\\s]*$");
+        }
+    }
+
+    public static class Left {
+        @BeforeTemplate
+        String before(String s) {
+            return StringUtils.left(s, 5);
+        }
+
+        @AfterTemplate
+        String after(String s) {
+            return s.substring(0, 5);
```

Comment:
Haven't checked if we need to deal with `null` here. Should be checked for all templates, I think.